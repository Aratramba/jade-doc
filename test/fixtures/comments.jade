//- @jadedoc
  name: foo

div.foo

  //- @jadedoc
    name: faa

  div.faa