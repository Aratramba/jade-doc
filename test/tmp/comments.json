[{"meta":{"name":"foo"},"file":"test/fixtures/comments.jade","source":"// foo","output":"<!-- foo-->"}]