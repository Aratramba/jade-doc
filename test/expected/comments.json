{
  "foo": {
    "jade": {
      "comment": "  name: foo",
      "type": "standalone-comment",
      "indent": 0,
      "root": "div.foo",
      "code": "div.foo\n  //- @jadedoc\n    name: faa\n  div.faa",
      "file": "./test/fixtures/comments.jade"
    },
    "name": "foo",
    "html": ""
  },
  "faa": {
    "jade": {
      "comment": "    name: faa",
      "type": "standalone-comment",
      "indent": 2,
      "root": "  div.faa",
      "code": "  div.faa\n",
      "file": "./test/fixtures/comments.jade"
    },
    "name": "faa",
    "html": ""
  }
}